query_optimizer/ExecutionGenerator.hpp

```diff
@@ -101,10 +101,10 @@ class ExecutionGenerator {
         query_handle_(DCHECK_NOTNULL(query_handle)),
         execution_plan_(DCHECK_NOTNULL(query_handle->getQueryPlanMutable())),
         query_context_proto_(DCHECK_NOTNULL(query_handle->getQueryContextProtoMutable())) {
+    query_context_proto_->set_query_id(query_handle->query_id());
```

Comment:
Since the `QueryHandle` has `query_id`, I was wondering why you need to add it in `QueryContext` proto?